terminal.go

```diff
@@ -62,6 +67,10 @@ func (t *Terminal) SetHistoryFile(f string) error {
 		log.Infof("No history file specified")
 		return nil
 	}
+	if t.capacity <= 0 {
+		log.Infof("No history capacity set, ignoring history file %s", f)
```

Comment:
I'm unsure if your logger support it, but for me here, it could be a warning (not info)

Reply:
logger does have warning level, yet if someone did explicitly pass 0 to disable, it shouldn't be a warning, it's just confirming what they did